src/modelAnalysis/topology/minimalCutSets.m

```diff
@@ -0,0 +1,93 @@
+function [output] = minimalCutSets(targets, varargin)
+% Computes minimal cut sets for paths/cycles/elementary modes with Berge algorithm
```

Comment:
The signature has to be changed to use optional arguments and not parameter/value pairs as required by @rmtfleming in #497